src/misc/macosx/SDL_sysurl.m

```diff
@@ -27,10 +27,10 @@
 SDL_SYS_OpenURL(const char *url)
 { @autoreleasepool
 {
-    NSString *nsstr = [NSString stringWithUTF8String:url];
-    NSURL *nsurl = [NSURL URLWithString:nsstr];
-    return [[NSWorkspace sharedWorkspace] openURL:nsurl] ? 0 : -1;
+	CFURLRef cfurl = CFURLCreateWithBytes(NULL, (UInt8 *) url, SDL_strlen(url), kCFStringEncodingUTF8,  NULL);
```

Comment:
This should probably be `const UInt8 *` rather than the mutable `UInt8 *`.

Reply:
Indentation is 4 spaces in SDL's codebase.

Reply:
I pushed a commit that fixes your two comments. I still have to add the iOS implementation (LSOpenCFURLRef is not available in iOS)